src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
Is this implementation supposed to fit in the current library or in the rewritten one? I thought it was supposed to go to this repo to fit in the rewritten library, but then it should not use the existing standard collection library. Instead:
- This file is in `strawman/collection` but in package `scala.collection.mutable`?
- This file uses the standard collection library...